bootstrap/terraform/vm.tf

```diff
@@ -0,0 +1,218 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "3.29.1"
+    }
+    azuread = {
+      source  = "hashicorp/azuread"
+      version = "2.30.0"
+    }
+  }
```

Comment:
Open to suggestions on pinning these to specific version or not doing so. Let me know!

Reply:
based on what I am reading [here](https://developer.hashicorp.com/terraform/tutorials/configuration-language/provider-versioning) +1 to pinning to a version

Reply:
It would be nice if we could automate bumping a pinned version to run in CI. Maybe that is a thing?

Reply:
Some convo around this at https://github.com/hashicorp/vault-plugin-auth-azure/pull/65#discussion_r1012302796. I think it's okay for TF to automatically download the latest since this is just for testing / development environments.